queue_services/entity-filer/flags.json

```
@@ -2,6 +2,7 @@
     "flagValues": {
         "enable-involuntary-dissolution": true,
         "namex-nro-decommissioned": true,
-        "enable-sandbox": false
+        "disable-nr-check": false,
+        "enable-sandbox": true
```

will not be using `disable-nr-check` anymore, use `enable-sandbox` instead of it and keep `enable-sandbox` value to false as default since we have to merge this back to main branch
```suggestion
        "enable-sandbox": false
```